src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends Figure {

```

Comment:
Don't begin class or method implementation with an empty line. Please remove the empty line at the start of the class definition.